homeassistant/components/sensor/swiss_hydrological_data.py

```diff
@@ -111,33 +88,35 @@ def __init__(self, hydro_data, measurement, value):
     def name(self):
         """Return the name of the sensor."""
         return "{0} {1} {2}".format(
-            self.hydro_data.data["name"],
-            self._measurement,
+            self._station,
+            self._condition,
             self._value)
```

Comment:
@fabaff What do you think about the way I name the sensors? Is that a good way?

Reply:
The "problem" is that the waterbody name is something I get from the API and if its down for some reason I cannot know the waterbody name when setting up the entities. Thats why I went for the station ID. The user still can set a friendly_name, right!?